src/shared/services/filter.service.ts

```diff
@@ -92,6 +96,9 @@ export class FilterService {
       },
     ];
 
-    this.store.dispatch(FilterActions.initializeFilters({ filterDefinitions }));
+    return await this.actionDispatcherService.dispatchAsync(
+      FilterActions.initializeFilters({ filterDefinitions }),
+      FilterActionEnum.loadFilters,
+    );
```

Comment:
do we need it here?

Reply:
I thought it would be a good idea to make it like this, we are initializing filter definitions here, we use this method for every other initialization part.